src/addons/editors/EditorContainer.js

```diff
@@ -259,10 +273,15 @@ const EditorContainer = React.createClass({
   },
 
   render(): ?ReactElement {
+    let divStyle = {
+      display: 'inline-block',
+      width: '200px',
+      height: '200px'
+    };
     return (
-        <div className={this.getContainerClass()} onKeyDown={this.onKeyDown} commit={this.commit}>
-        {this.createEditor()}
-        {this.renderStatusIcon()}
+        <div style={divStyle} className={this.getContainerClass()} onBlur={this.handleBlur} onKeyDown={this.onKeyDown} commit={this.commit}>
```

Comment:
I have a editor renderer with date picker. As the data picker popup is a single div and when try to open the popup, the onBlur will be called and this.commit will prevent the popup. Can we make this onBlur configable?